src/pages/email.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import { Button, Container, Input, Text, Textarea } from '@chakra-ui/react';
+
+const Email = () => {
+  const [loading, setLoading] = useState(false);
+  const [message, setMessage] = useState<string>('');
+  const [emailText, setEmailText] = useState('');
+  const [emailRecipients, setEmailRecipients] = useState([]);
+
+  const handleEmailSend = async () => {
+    setLoading(true);
+    setMessage('');
+    try {
+      const response = await fetch('/api/sendEmail', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          to: emailRecipients,
+          subject: 'Hello World',
```

Comment:
Could we add another Input/TextArea in the form and get the subject from there?

Reply:
done